roles/openshift_aws/tasks/provision.yml

```diff
@@ -29,3 +29,20 @@
     search_regex: OpenSSH
   with_items: "{{ instancesout.instances }}"
   when: openshift_aws_wait_for_ssh | bool
+
+- when: openshift_aws_restart_after_creation | default(false) | bool
+  block:
+  - name: restart host
+    ec2:
+      instance_id: "{{ item.instance_id }}"
+      state: restarted
+      region: "{{ openshift_aws_region }}"
+    with_items: "{{ instancesout.instances }}"
+  - name: wait for ssh to become available
+    wait_for:
```

Comment:
I'd use wait_for_connection which actually waits for SSH transport to be alive. It seems to be the preferred way to wait for a host to become available again.

Reply:
Got an example?

Reply:
Something already used in the openshift-ansible?

Reply:
Ho, just this? https://github.com/openshift/openshift-ansible/blob/master/playbooks/openstack/openshift-cluster/provision.yml#L20 Nice

Reply:
@sdodson done, think it make sense the same check on the line 24?